velox/dwio/parquet/reader/ParquetReader.h

```diff
@@ -20,7 +20,14 @@
 #include "velox/dwio/common/ReaderFactory.h"
 #include "velox/dwio/parquet/reader/duckdb/Allocator.h"
 #include "velox/dwio/parquet/reader/duckdb/InputStreamFileSystem.h"
+#ifdef __clang__
```

Comment:
This is a lot of text. Would you add a comment explaining what's going on?

Reply:
This is a standard approach to allow the compiler to ignore warnings in a block of code. I will add another pragma that should simplify this.
Another usage is here https://github.com/facebookincubator/velox/blob/main/velox/experimental/codegen/vector_function/GeneratedVectorFunction-inl.h#L120